apply/templates/apply/dashboard_single_fund.html

```diff
@@ -53,6 +54,15 @@ <h2 class="govuk-heading-l">
                 {% if round["is_past_submission_deadline"] %}
                     {{ round_closed_warning(fund["fund_data"]["name"], round["round_details"]["title"], round["round_details"]["deadline"]) }}
                 {% endif %}
+                {% for application in round["applications"] %}
+                    {% if application.status == 'CHANGES_REQUESTED' %}
```

Comment:
Only a tiny problem...

In the situation where you have mroe applications with Changes requestes, you'd display the banner multiple time.

I think one time is enough. So I'd change the logic to display it if there is any applicationwith that status.

I did not test it, but perhaps this will work in jinja..
`{% if 'CHANGES_REQUESTED' in [application.status for application in round["applications"]] %}`

Reply:
You're right, I didn't realise that it's a page where multiple applications are displayed, I assumed it being a 'detail' page for a single application. Changed it now, so it's displayed only once.